test/controllers/benefits/RemoveCompanyBenefitControllerSpec.scala

```diff
@@ -574,12 +605,12 @@ class RemoveCompanyBenefitControllerSpec
         val SUT = createSUT
         val employmentId: String = "1234"
         val endedCompanyBenefit =
-          EndedCompanyBenefit("Accommodation", "Before 6th April", None, "No", None)
+          EndedCompanyBenefit("Accommodation", stopDateFormatted, None, "No", None)
```

Comment:
Why use both stopDateFormatted and stopDate when they look to be the same?

Reply:
nope, one is 6 April 2022 the other one is  "Before 6th April"

Reply:
Ahh